### PR TITLE
laf-1-ref-box

### LINKED ISSUE #115: feature/laf-1-ref-box
## Problem description
The Item-View must provide reference box with the connection for the featured services. The reference box should looks like this 
image

- Create a new component `ds-item-ref-table` that will be rendered in the item view component `ds-item-page`. This new component will send a request to the BE to fetch data (related to citations) - refbox API.
- Create a new component `ds-item-citation` that will display citation content. This component should be rendered in the item view.
- Add buttons to share the data on the social media (FB, Twitter).

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  12.8  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  8+8  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Problem description
Check the issue.

## Problems
- Problem: If the Collection doesn't has defined Entity Type and the user creates the Item in that Collection, the created Item has created metadata field `dspace.entity.type` but without value. This state throws error when the Item is harvesting via OAI-PMH - method GetRecord. The error is thrown in the `DSpaceAtLeastOneMetadataFilter`.
**Solution:** Updated WorkspaceItemServiceImpl to do not add empty metadata field `dspace.entity.type` to the Item.

### CODE REVIEW

File: src/app/item-page/clarin-ref-box/clarin-ref-box.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ClarinRefBoxComponent } from './clarin-ref-box.component';
+
+/**
+ * The test class for `ClarinRefBoxComponent`
+ */
+describe('ClarinRefBoxComponent', () => {
+  // TODO make tests
+  // let component: ClarinRefBoxComponent;
```
Comment (mrovnanik) [inline src/app/item-page/clarin-ref-box/clarin-ref-box.component.spec.ts:9]: Waiting for whom?

File: src/app/item-page/clarin-ref-citation/clarin-ref-citation.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ClarinRefCitationComponent } from './clarin-ref-citation.component';
+
+/**
+ * The test class for the ClarinRefCitationComponent
+ */
+describe('ClarinRefCitationComponent', () => {
+  // TODO create tests
+  // let component: ClarinRefCitationComponent;
```
Comment (mrovnanik) [inline src/app/item-page/clarin-ref-citation/clarin-ref-citation.component.spec.ts]: When are we expected to finish these tests?

**Review by mrovnanik — CHANGES_REQUESTED** [review]
Tests are missing

File: src/app/item-page/clarin-ref-citation/clarin-ref-citation.component.spec.ts (hunk 2 above)
Comment (milanmajchrak) [reply src/app/item-page/clarin-ref-citation/clarin-ref-citation.component.spec.ts]: Removed the tests files

**Pull request merged**